src/common/droid-util.c

```diff
@@ -1580,14 +1580,8 @@ void pa_droid_stream_unref(pa_droid_stream *s) {
         pa_xfree(s->output);
     } else {
         pa_log_debug("Destroy input stream %p", (void *) s);
-        set_active_input(s->module, NULL);
-        pa_mutex_lock(s->module->input_mutex);
         pa_idxset_remove_by_data(s->module->inputs, s, NULL);
-        if (s->input->stream) {
-            s->input->stream->common.standby(&s->input->stream->common);
-            s->module->device->close_input_stream(s->module->device, s->input->stream);
-        }
-        pa_mutex_unlock(s->module->input_mutex);
+        input_stream_close(s);
```

Comment:
should this too check that close input quirk?

Reply:
Nope, as this is happening when the whole stream is torn down.